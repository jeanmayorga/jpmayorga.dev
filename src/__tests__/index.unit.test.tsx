import React from 'react';
import renderer from 'react-test-renderer';

import Home from './../pages/index';

describe('Home page', () => {
  test('renders', () => {
    const component = renderer.create(<Home />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
