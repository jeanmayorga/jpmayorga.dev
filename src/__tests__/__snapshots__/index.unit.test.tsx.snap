// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders 1`] = `
<div
  className="w-screen h-screen"
>
  <main
    className="w-full h-full flex justify-center items-center"
  >
    <div>
      <p
        className="text-black font-bold mb-4"
      >
        Welcome to your NextJS application!
      </p>
      <p
        className="text-black flex justify-around"
      />
    </div>
  </main>
</div>
`;
